storybookjs/storybook
# Add ability to use image snapshots to addon-storyshots

Issue: https://github.com/storybooks/storybook/issues/1781

## What I did
- Ability to generate image snapshots from stories using addon-storyshots.

## How to test
- Start fresh & bootstrap everything
- Go to _cra-kitchen-sink_
- Run `yarn run test-image-snapshots`

Is this testable with jest or storyshots?
- Not sure 

Does this need a new example in the kitchen sink apps?
- Yep done.

Does this need an update to the documentation?
- Maybe we should say something about it in the doc as it may persuade people to use storybook ? But I don't think there is a word about storyshots in the doc yet.
